manifests/deis-router-rc.yaml

```diff
@@ -23,3 +23,7 @@ spec:
         ports:
         - containerPort: 80
           hostPort: 80
+        - containerPort: 443
+          hostPort: 443
+        - containerPort: 2222
```

Comment:
Do we use `2222` for anything beside builder? Wondering how (if?) manifests are merged together on the fly and if the builder could own that port opening instead, as in instruct router to open it up. Probably a bit too complicated for now tho.